README.md

```diff
@@ -52,216 +18,126 @@ microservice architectures efficiently.
+1. First download the CLI tool(APICTL) and the microgateway distribution from the  
 [github release page](https://github.com/wso2/product-microgateway/releases)
-and extract it to a folder of your choice.
+and extract them to a folder of your choice.
+  * [CLI (APICTL)](https://github.com/wso2/product-microgateway/releases)
+  * [Microgateway Distribution](https://github.com/wso2/product-microgateway/releases)
+  
+CLI tool extracted location will be referred as `CLI_HOME` and Microgateway distribution extracted location would be 
+referred as `MG_HOME`.
 
-1. Using your command line client tool add the 'bin' directory of the extracted folder to your PATH variable.
+2. Using your command line client tool add the 'CLI_HOME' folder to your PATH variable.
 ```
-export PATH=$PATH:<TOOLKIT_EXTRACTED_LOCATION>/bin
+export PATH=$PATH:<CLI_HOME>
 ```
 
-3. We are now ready to execute the Microgateway toolkit commands to initialize and build our Microgateway. Let's create 
-our first project with name "petstore" by adding the [open API definition](https://petstore.swagger.io/v2/swagger.json) of the petstore . You can do that by executing the following command using your command line tool.
+3. Let's create our first project with name "petstore" by adding the [open API definition](https://petstore.swagger.io/v2/swagger.json) of the petstore . You can do that by executing the following command using your command line tool.
 ```
-micro-gw init petstore -a https://petstore.swagger.io/v2/swagger.json
+apictl init petstore -oas https://petstore.swagger.io/v2/swagger.json
 ```
 
 4. The project is now initialized. You should notice a directory with name "petstore" being created in the location 
 where you executed the command. 
 
  
-5. Next, Lets build the project and create a microgateway docker image.
+5. Now lets deploy the microgateway on docker by executing the docker compose script inside the `MG_HOME`. Navigate to `MG_HOME` and execute the following command
 ```
-micro-gw build petstore --docker-image petstore:v1 --docker-base-image wso2/wso2micro-gw:3.1.0
+docker-compose up -d
 ```
 
-Once the build is successful microgateway docker image will be created with name "petstore:v1"
+Once containers are up and running, we can monitor the status of the containers using the following command
 
-6. Now gateway can be started using the built docker image
-Execute the command below to run the Microgateway for our Petstore project.
+```
+docker ps | grep mg-
+```
+
+6. Now let's deploy our first API to Microgateway using the project created in the step 3. Navigate to the location where the petstore project was initialized.
+Zip the petstore project and create the file petstore.zip
+Execute the following command to deploy the API in the microgateway
 
 ```
-docker run -d -p 9090:9090 -p 9095:9095 petstore:v1
+apictl mg deploy --host https://localhost:9095 --file "/Users/viraj/Documents/petstore.zip"  -u admin -p admin
```

Comment:
Either -k option needs to be mentioned or else we should advise on how to add the cert